demo/combo-box-basic-demos.html

```diff
@@ -115,8 +179,8 @@ <h3>Lazy Loading</h3>
               };
               var index = params.page * params.pageSize;
               const url = 'https://demo.vaadin.com/demo-data/1.0/filtered-countries?index=' + index +
-                          '&count=' + params.pageSize +
-                          '&filter=' + params.filter;
+              '&count=' + params.pageSize +
```

Comment:
2 spaces of indentation would look better here than no indentation

Reply:
Done